oh-my-zsh.sh

```diff
@@ -120,17 +120,18 @@ fi
 
 if [[ "$ZSH_DISABLE_COMPFIX" != true ]]; then
   source "$ZSH/lib/compfix.zsh"
-  # If completion insecurities exist, warn the user
-  handle_completion_insecurities
   # Load only from secure directories
-  compinit -i -C -d "$ZSH_COMPDUMP"
+  compinit -i -d "$ZSH_COMPDUMP"
+  # If completion insecurities exist, warn the user
+  handle_completion_insecurities &|
```

Comment:
hey, do you know where I can find a list of those very useful keywords ? (tried some googling as well as `man zshall`, haven't found a decent list :/)

Reply:
Which keywords are you exactly talking about?

Reply:
the `&|`, and I've today also discovered `&!`

Reply:
I always find that information in `man zshall`!

Reply:
That's weird.. On my mac somehow it's not there (`man zshall | rg --fixed-strings '&!'`)

Found it under `man zshbuiltins` though!

Thanks for your time and that PR 🙂